### DISCUSSION TITLE
[Solved] Feature Request to Hide Usernames

### DISCUSSION THREAD

**Discussion by yontekh** [Chatterino/chatterino2#3284] [Q&A] [answered]
I would like to be able to hide usernames completely.

I watch big streamers with chats that fly by at hyper speed. I don't care who says what cause interaction is impossible anyway.
I just wanna see Kappa's and LULs fly by so I can see the chats general response while watching Mr Strimmer.

Thanks for considering my request, I hope you find time to add the feature.

**Comment by Mm2PL** [comment, marked as answer]
You can regex everyone's usernames with Nicknames to nothing.

**Settings -> Nicknames -> Add**

![2021-10-12_21-31](https://user-images.githubusercontent.com/25011746/137017631-c9c147f3-cfdc-40cc-b239-cfb24035dc5d.png)

Please note that the *Nickname* field is empty.

**Reply by yontekh** [reply]
Thanks, this works great!
